medusa/indexers/tvdbv2/tvdbv2_api.py

```diff
@@ -340,7 +341,8 @@ def _parse_images(self, sid):
                 if image_type not in _images:
                     _images[image_type] = {}
 
-                images = self.series_api.series_id_images_query_get(sid, key_type=image_type)
+                images = self.series_api.series_id_images_query_get(sid, key_type=image_type,
+                                                                    accept_language=self.config['language'])
```

Comment:
what if `images` returns Italian poster and an English poster? Which will be used? shouldn't we put meta data language first?